.github/workflows/code-quality.yml

```
@@ -5,16 +5,20 @@ on:
   push:
     branches:
       - master
+    paths:
+      - "ruff.toml"
   pull_request:
+    paths:
+      - "ruff.toml"
```

You can remove this to run this action every time. 
The ruff configuration has been moved to `pyproject.toml`
```suggestion
  pull_request:
```